src/components/Draggable/FilterDraggable.tsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react'
+import { DndProvider } from 'react-dnd'
+import { HTML5Backend } from 'react-dnd-html5-backend'
+
+import { RealFilterDraggable } from './RealFilterDraggable'
+
+export interface FilterDraggableProps<T> {
```

Comment:
Could extend `DefaultDraggableProps` to dont repeat some properties?

Reply:
Sure !